src/placement/tests/place_obj_common.c

```diff
@@ -83,6 +83,22 @@ reint_check(struct pl_obj_layout *layout, struct pl_obj_layout *temp_layout,
 		uint32_t *spare_tgt_ranks, uint32_t *shard_ids, int num_reint,
 		uint32_t curr_fail_tgt)
 {
+<<<<<<< HEAD
+	int i, j;
+	int temp_i;
+	int rebuilding;
+	int num_reint_found;
+	uint32_t shard_idx;
+	uint32_t target;
+	uint32_t original_target;
+	uint32_t reint_target;
+	struct pl_obj_shard curr_shard;
+
+	D_ASSERT(num_reint >= 0 && num_reint < 2);
+	num_reint_found = 0;
+	temp_i = 0;
+	i = 0;
+=======
```

Comment:
(style)  spaces required around that '==' (ctx:ExO)
(style)  spaces required around that '==' (ctx:OxO)
(style)  spaces required around that '==' (ctx:OxO)
(style)  spaces required around that '=' (ctx:OxE)